gtk/TreeModelFilter.custom

```diff
@@ -68,7 +68,7 @@
 			IntPtr[] native_types = new IntPtr [types.Length];
 			for (int i = 0; i < types.Length; i++)
 				native_types [i] = types [i].Val;
-			GCHandle gch = GCHandle.Alloc (func);
+			GCHandle gch = func != null ? GCHandle.Alloc (func) : (GCHandle)IntPtr.Zero;
 			gtk_tree_model_filter_set_modify_func (Handle, n_columns, native_types, GtkSharp.TreeModelFilterModifyFuncWrapper.NativeDelegate, (IntPtr) gch, GLib.DestroyHelper.NotifyHandler);
```

Comment:
Do we want to pass a null destroy callback when func is null? It'd be consistent with everything else, but not a big deal

Reply:
Also, s there a particular reason why we want to keep passing `GtkSharp.TreeModelFilterModifyFuncWrapper.NativeDelegate` to native when the `func` is null?

Reply:
DestroyNotify handles null gchandle, so we're ok:
```
static void ReleaseGCHandle (IntPtr data)
		{
			if (data == IntPtr.Zero)
				return;
			GCHandle gch = (GCHandle) data;
			gch.Free ();
		}

```

Reply:
Hmmm, maybe not. I can send in null. This was custom bound, didn't verify integrity of it